### PR TITLE
Update learn-consensus.md

### PR DESCRIPTION
**Pull request by DrW3RK**
What SASSAFRAS stands for

### CODE REVIEW

File: docs/learn/learn-consensus.md [hunk 1]
```diff
@@ -124,7 +124,8 @@ For more details on BABE, please see the
 
 ### BADASS BABE: SASSAFRAS
 
-BADASS BABE is an extention of BABE and acts as a constant-time block production protocol.
+Stands for Semi Anonymous Sortition of Staked Assignees For Fixed-time Rhythmic Assignment of Slots.
```
Comment (salmad3) [inline docs/learn/learn-consensus.md]:
```suggestion
SASSAFRAS (Semi Anonymous Sortition of Staked Assignees For Fixed-time Rhythmic Assignment of Slots)
```

File: docs/learn/learn-consensus.md [hunk 2]
```diff
@@ -124,7 +124,8 @@ For more details on BABE, please see the
 
 ### BADASS BABE: SASSAFRAS
 
-BADASS BABE is an extention of BABE and acts as a constant-time block production protocol.
+Stands for Semi Anonymous Sortition of Staked Assignees For Fixed-time Rhythmic Assignment of Slots. 
+It is an extention of BABE and acts as a constant-time block production protocol.
```
Comment (salmad3) [inline docs/learn/learn-consensus.md]:
```suggestion
(aka SASSY BABE or BADASS BABE), is an extension of BABE and acts as a constant-time block production protocol.
```

**Review by salmad3 — CHANGES_REQUESTED** [review]
Follows the flow of the doc

File: docs/learn/learn-consensus.md (hunk 2 above)
Comment (profradha) [reply docs/learn/learn-consensus.md]: I'm unsure about the origins of SASSY BABE or BADASS BABE, but if they are appropriate, please include them

**Review by salmad3 — APPROVED** [review]

**Pull request merged**